roles/confluent.kafka_connect/tasks/main.yml

```diff
@@ -57,6 +57,17 @@
     force: true
   when: installation_method == "archive"
 
+# Non-Archive File deployments need to create SystemD service units when the user changes the service name
+- name: Copy Kafka Connect Service from default install to system
+  copy:
+    src: "/lib/systemd/system/{{kafka_connect_default_service_name}}.service"
```

Comment:
this is hardcoded for ubuntu it looks like

Reply:
Fixed